library/src/main/java/com/firebase/ui/auth/google/GoogleAuthProvider.java

```diff
@@ -93,7 +82,7 @@ public void onResult(Status status) {
     public void login() {
         if (mGoogleApiClient.isConnected()) {
             Intent signInIntent = Auth.GoogleSignInApi.getSignInIntent(mGoogleApiClient);
-            mActivity.startActivityForResult(signInIntent, GoogleActions.SIGN_IN);
+            ((Activity)getContext()).startActivityForResult(signInIntent, GoogleActions.SIGN_IN);
```

Comment:
Curious about the reasoning for the change here, I'm assuming this functioned before, so why the change?